### ISSUE TITLE
Methods to indulge students in CodeDocs activities-Discussion

### ISSUE THREAD

**Issue by samyak360** [CodeDocsJECRC/community-committee#15]
What should be done to increase the active participation of students in CodeDocs activities?

**Comment by maddhruv** [comment]
The major problem we faced in the past was this. :arrow_up: 
Actually, the best solutions one can get for such a community are - Slack, GitHub, Trello, etc. but most of the included audience is not active or don't know about these tools, which is a very weak point.

There could be two solutions for this - 
- get into more familiar tools - WhatsApp kinda - which are handy to everyone - but is the worst idea we could even think of
- the better way - introduce people to these heavenly tools in #13 and get them used to - like we should strictly stick to them - no discussions everywhere, would do the trick I guess.

**Comment by accakks** [comment]
- **Open space every weekend:** An informal discussion in a classroom, no agenda, anyone can speak up. We need to start conversations in first 1-2 open spaces. sharing opportunities, making them realise that even from JECRC, they can achieve big things etc.
- **Interesting activities and contests** 
- **Social media** Amazing social media channels inspires people.  #17 
- **Interesting names & graphics for every other activity** I have been into marketing. These things create a huge impact. #17

**Comment by accakks** [comment]
We can include tools and all to be used in orientation presentation. @maddhruv

**Comment by maddhruv** [comment]
> Interesting names & graphics for every other activity

can you elaborate more of this part
what kind of name are you referring to?

**Comment by accakks** [comment]
Like using cool words like 'hackers' and cool graphics on social media.
Using attractive names for sessions. 

Like there was some talk named something like `.....combat Man-in-the-middle attack.`

So If we wanna have a session for security and stuff, We can name it like that and have nice graphic for the same. 
Content is effective too. Including hacker references from movies etc etc etc

This is one of the awesome ways to attract students!

**Comment by amritsrivastava** [comment]
We can motivate students for using these techs tools by telling them the benefits they get in their career.

**Comment by Akshaygagrani** [comment]
I think we must work upon the past mistakes the major reason of the inactiveness of the student with code-docs in the previous session was the irregularity and the failure of not providing the full course accomplishment and regularity with all work and assignment, simultaneously on our side we know we were not able to give the proper time and knowledge to them because of college permissions, activities and our own packed schedule, so I suggest that all the enthusiast students right now do not have the proper knowledge and idea of how important role such a society plays for building up the carrier.

**Lot talking about the problems** actually i thought about a way to get rid out of this irregularity and loose connection with students the holiday is the best time when most of the students spend time over themselves to learn and nourish the knowledge so we can create webinar or go live on youtube(simply by creating a youtube channel of CODEDOCS) and share the links to the college students as well as outside of the campus **it will increase our outreach also through it, this will create a good and faithful image of the society in front of everyone and most importantly, then after peoples can trust upon us and on our promises (which we were in-capable of previously)** and students+new cumors will come to know about us, initially we will not hit the topics like intro to git or etc. we will start the basic talk like the importance of coding environment and group+intro to git this will create more easy for us to impart the knowledge with eg +we will add the link on our website and apps of the same so that they can watch that any time and the assignments can also be made regular through it. 

The college will always keep organizing the activities and keep the crowd busy, and will never let us fulfill our aim. **We ourselves have to make out the time for ourselves and the society so this session initially, and connect, indulge enthusiastic students with us.**

**Comment by accakks** [comment]
@Akshaygagrani
True. This is the reason we're planning for proper orientation before we start, so that first students realise importance of community , how to give time, how it works etc. 

+ We're planning to complete classes in odd sem, & even sem would be for advanced topics & activities. Since this semester is less disturbed, it would be good to have classes now. 

I like the YouTube idea. We'll upload the orientation on YouTube channel, for starters. And we might also go live or provide notes of tje class anyway. If students miss, they should be able to recover and learning should not stop. 

Next about git, I agree, as a newbie's perspective it seems very intimidating. 

We need to have that at start though as that's where projects are hosted, but I am thinking of dividing git to two parts to make it easier + I have found some nice sources & blogs which takes things in easier manner & implement same in teaching. 

But even after this, I know that some part would always be intemidating. It has been , for all of us, everytime we learn something new. 

But to support through process, I have also suggested openspace concept. Where open discussions happen , maybe once in two weeks. I'm very excited about this part.

You can check on other issues about it. 

When students participate more & engage in discussions, community will grow & improve.

**Comment by Akshaygagrani** [comment]
I agree that this semester is odd and the work load will be less we can plan the classes easily further I would like to add that we need recognition in our upcoming 2nd and 1st year because if we do not get active persons from upcoming 2nd year(this have to be our first main goal in order to rise the group) else the group will not be able to become eternal and indulge the 1st year because they won't take initiative for such a thing which do not have any contribution of their next senior,and eventually it will bring the end. 

  so as a new beginning let's​ start by telling them why they actually need CODEDOCS and how to dive into open source and take the use of the society similarly the connectivity through the digital platform and mandatory online assignments will definitely impart the knowledge and fulfill our aim so,

 I just wanna vote that we must start thinking of becoming a constant and trustful face for everyone where all the beginners can rely on in order to solve their confusions that's all they need from us and then after as the time passes by we raise our standard to next level. That's all I wanna contribute for this issue 🙂😅.

And I really appreciate the part of open space Discussion it will definitely help to become CODEDOCS a problem saviour ​ for everyone.But we will through this spell at the moment when everyone becomes familiar and the knowledge gap also gets little filled+the student start become connected to us.

**Comment by accakks** [comment]
@Akshaygagrani If you would like to give any suggestion for orientation -> #13
